src/client/components/segment-bubble/segment-bubble.tsx

```diff
@@ -81,13 +83,49 @@ export class SegmentBubble extends React.Component<SegmentBubbleProps, SegmentBu
     }
   }
 
+  getReplacedUrl(label: string) {
+    var { urls, datum } = this.props;
+    var idx = 0;
+    if (urls.length !== 1) {
+      const nestIndex = (datum['__nest'] as number) - 1;
+      idx = nestIndex;
+    }
+    var url = urls[idx];
+    return url.replace('%s', label);
```

Comment:
it is better to do `.replace(/%s/g, label)` it will replace all occurrences of `%s` which is usually more expected. Always be weary of having a string as the first argument in `replace()`, ask yourself is that what I really want?